src/main/java/vention/pages/PreLoginPage.java

```diff
@@ -0,0 +1,29 @@
+package vention.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class PreLoginPage extends BasePage {
+  @FindBy(xpath = "//button[@data-testid='login-button']")
+  private WebElement loginButton;
+
+  @FindBy(xpath = "//button[@data-testid='register-button']")
+  private WebElement registerButton;
+
+  @Override
+  public String getRelativePath() {
+    return "/nx/b/account/en/usa/account/pre-login";
```

Comment:
I'd recommend to move the string to a constant variable in this class. This comment relates to all the similar cases.

We don't usually hardcode values inside the methods, we create constant variables for such cases